includes/class-events-store.php

```diff
@@ -882,6 +879,315 @@ public function count_events_by_status( $status ) {
 
 		return (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(ID) FROM {$this->get_table_name()} WHERE status = %s", $status ) ); // Cannot prepare table name. @codingStandardsIgnoreLine
 	}
+
+	/*
+	|--------------------------------------------------------------------------
+	| New event's store methods. The above may be deprecated in the future.
+	| But notably, the below is also internal-usage only. See comments about alternatives.
```

Comment:
"internal" meaning just within the "cron control" plugin internals specifically, not even the rest of mu-plugins. Any caller should be going through Event/Events.

Would definitely be ideal to be truly private, but kind of hard to accomplish within the existing Singleton structure. In the future once I'm able to deprecate out the public methods in the singleton, I envisioned something more strict. At which point I'd be pretty comfortable dropping these "fake private" methods since it was pretty clear one was not supposed to use these.